<ui version="4.0" >
 <class>WPicViewerBase</class>
 <widget class="QDialog" name="WPicViewerBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>558</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>100</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Picture Viewer</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>1</x>
     <y>1</y>
     <width>661</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <item row="0" column="0" colspan="5" >
     <widget class="QLabel" name="pxlPixmap" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>32767</width>
        <height>32767</height>
       </size>
      </property>
      <property name="scaledContents" >
       <bool>false</bool>
      </property>
      <property name="alignment" >
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap" >
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="QPushButton" name="btnFirst" >
      <property name="text" >
       <string>First</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" >
     <widget class="QPushButton" name="btnPrevious" >
      <property name="text" >
       <string>Previous</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" >
     <widget class="QLabel" name="txtItems" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>0</width>
        <height>24</height>
       </size>
      </property>
      <property name="text" >
       <string>No File</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap" >
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="3" >
     <widget class="QPushButton" name="btnNext" >
      <property name="text" >
       <string>Next</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4" >
     <widget class="QPushButton" name="btnLast" >
      <property name="text" >
       <string>Last</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" >
     <widget class="QPushButton" name="btnOpen" >
      <property name="text" >
       <string>Open</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2" >
     <widget class="QPushButton" name="btnSave" >
      <property name="text" >
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3" >
     <widget class="QPushButton" name="btnClose" >
      <property name="text" >
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
